java-frontend/src/main/java/org/sonar/java/resolve/Resolve.java

```diff
@@ -426,12 +433,27 @@ private Resolution findMethod(Env env, JavaType callSite, JavaType site, String
     return bestSoFar;
   }
 
+  private Resolution lookupInScope(Env env, JavaType callSite, JavaType site, String name, List<JavaType> argTypes, List<JavaType> typeParams,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00107)